codegen/transformers.go

```diff
@@ -43,10 +43,6 @@ func defaultGoTypeToSchemaType(v reflect.Type) (schema.ValueType, error) {
 		return schema.TypeBool, nil
 	case reflect.Int, reflect.Int8, reflect.Int16, reflect.Int32, reflect.Int64,
 		reflect.Uint, reflect.Uint8, reflect.Uint16, reflect.Uint32, reflect.Uint64:
-		// need to check for duration here, as Go time.Duration is int64
-		if v == reflect.TypeOf(time.Duration(0)) {
-			return schema.TypeTimeInterval, nil
-		}
```

Comment:
Oh I see it's also removed here. Could this break an existing plugin by any chance?